packages/perennial-vaults/contracts/balanced/BalancedVaultDefinition.sol

```diff
@@ -61,20 +62,30 @@ contract BalancedVaultDefinition is IBalancedVaultDefinition {
     uint256 private immutable weight0;
     uint256 private immutable weight1;
 
+    /**
+     * @param controller_ The controller contract
+     * @param targetLeverage_ The target leverage for the vault
+     * @param maxCollateral_ The maximum amount of collateral that can be held in the vault
+     * @param marketDefinitions_ The market definitions for the vault
+     * @param previousImplementation_ The previous implementation of the vault. Set to address(0) if there is none
```

Comment:
I think we should add a dev comment to specify that `previousImplementation_` is essentially an optional feature that deploy scripts can utilize to give extra protections against parameter errors during an upgrade process. We don't want this to seem like it's a fool proof way to verify every upgrade (since you can simple not provide the previous implementation if you don't want to)